src/app/actions/users.ts

```
@@ -1,6 +1,6 @@
 import { type ApiUser } from '@/services/users'
 import { type AddBankAccountPayload, BridgeEndorsementType, type InitiateKycResponse } from './types/users.types'
-import { type User } from '@/interfaces'
+import { type CounterpartyUser } from '@/interfaces'
```

ESLint

[error] '`@/interfaces`' import is restricted from being used. Import from a specific file instead of the '`@/interfaces`' barrel — barrels force the bundler to load every re-export and hurt build perf. See CLAUDE.md.

(no-restricted-imports)